### PR TITLE
Feature/back button with saved value

### PR DESCRIPTION
**Pull request by ektasawant**
Fixes PXBLUE-2210 .

#### Changes proposed in this Pull Request:
-  Enable back button on Self registration and invitation-based registration form
-  Save email and password through out registration process
- Rename Back button label to Cancel in Reset password form

#### To Test:
-

### CODE REVIEW

File: login-workflow/src/translations/french.ts [hunk 1]
```diff
@@ -8,6 +8,7 @@ export const pxbAuthFrenchTranslations: PxbAuthTranslations = {
         OKAY_BUTTON: `D'accord`,
         DONE_BUTTON: 'Done',
```
Comment (joebochill) [inline login-workflow/src/translations/french.ts:9]: Can you change 'Done' to 'Fini'

**Review by joebochill — COMMENTED** [review]
This touches more things than I was expecting it would...I'm going to let @emclaug2 weigh in on this.

**Comment by emclaug2** [comment]
> This touches more things than I was expecting it would...I'm going to let @emclaug2 weigh in on this.

I took a look at this, since we have components for the change-password and email input fields, they needed to be altered so that we could read from a state service.  I'm going to run through the registration flow a couple times and make sure this is the expected behavior.

**Comment by emclaug2** [comment]
One thing I noticed is that the password and email field's retain their original values after registering is complete. I would expect all those values to be cleared from the state.

**Comment by ektasawant** [comment]
> One thing I noticed is that the password and email field's retain their original values after registering is complete. I would expect all those values to be cleared from the state.

Clearing saved registration email and password after successful registration.

**Comment by jeffvg** [comment]
looks like Rename Back button label to Cancel in Reset password form needs updated. Other than that looks good

**Comment by ektasawant** [comment]
> looks like Rename Back button label to Cancel in Reset password form needs updated. Other than that looks good

Got this suggestion in last demo session. Please let me if I need to rename it again to `Back` label?

**Review by emclaug2 — APPROVED** [review]
Changes look good to me now; state is reset after completing or exiting registration.

**Comment by joebochill** [comment]
This looks correct to me based on the comments from the last review (The label should be 'Cancel' because the reset password screen is only accessed via deep link, so there is no 'Back')

@jeffvg please elaborate if you were expecting something else there

**Pull request merged**